utils/data_processing/shell_scripts/polyethylene_film.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
I suspect this file shouldn't be here and neither this folder, right?

Reply:
somehow I didn't pay attention to it when the folder was introduced but I think it is the user's responsibility to know how to use the command. There are examples in the documentation already in case they are looking for guidance